core/src/main/java/org/apache/accumulo/core/metrics/MetricsDocGen.java

```diff
@@ -52,21 +52,27 @@ void pageHeader() {
     doc.println("---\n");
     doc.println("<!-- WARNING: Do not edit this file. It is a generated file"
         + " that is copied from Accumulo build (from core/target/generated-docs) -->\n");
-    doc.println("Below are the metrics used to monitor various components of Accumulo.\n");
+    doc.println(
+        "Below are the metrics used to monitor various components of Accumulo. Metrics emitted by Accumulo should"
+            + " contain the folowing tags: 'instance.name', 'resource.group', 'process.name', 'host' and 'port'. Metrics"
+            + " emitted by Accumulo may contain additional tags where we think it makes sense to capture per-object metrics,"
+            + " for example on a table or tablet basis in the ScanServer and TabletServer, or on a per-queue basis in the"
+            + " CompactionCoordinator.\n");
   }
 
   void generateTableOfContents() {
     doc.println("## Table of Contents\n");
-    for (var category : Metric.MetricCategory.values()) {
+    for (var category : Metric.MetricDocSection.values()) {
       String sectionId = generateSectionId(category.getSectionTitle());
       doc.println("- [" + category.getSectionTitle() + "](#" + sectionId + ")");
     }
     doc.println();
   }
 
-  void generateCategorySection(Metric.MetricCategory category, String sectionTitle) {
+  void generateCategorySection(Metric.MetricDocSection category, String sectionTitle,
```

Comment:
It could make  sense to just accept the `Metric.MetricDocSection category` as the only argument here and use it to get the title and description. That would make for a bit cleaner code probably. Also, could update the name of `category` in the method and var names since its changed to section.

Reply:
Took care of this in 27cd47b